src/main/java/bio/terra/janitor/service/cleanup/FlightSubmissionFactoryImpl.java

```diff
@@ -32,4 +40,16 @@ public UnsupportedCleanupFlight(FlightMap inputParameters, Object applicationCon
       addStep(new FinalCleanupStep(janitorDao));
     }
   }
+
+  /** A Flight to cleanup a GoogleBucket. */
+  public static class GoogleBucketCleanupFlight extends Flight {
```

Comment:
nit: maybe split out the cleanup flights into their own files? I think it might be nice to have a subdirectory `service/cleanup/flights` to hold all of our Flights and Steps.

Reply:
All steps are already under `service/cleanup/flights`. Also moved to flight there